pyproject.toml

```diff
@@ -20,11 +20,12 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-    "cosmology>=2022.10.9",
+    "cosmology.api>=0.3.0",
     "healpix>=2022.11.1",
     "healpy>=1.15.0",
     "numpy>=2.1.1",
     "transformcl>=2022.8.9",
+    "typing_extensions>=4.13.2",
```

Comment:
@ntessore I've had to add dependency as it is required in the latest `cosmology.api` release (added in https://github.com/cosmology-api/cosmology.api/pull/120). Should it be a dependency of `cosmology.api` instead?

Reply:
I think that's a bug that was manually introduced there, let me fix it.

Reply:
Something wrong with `0.3.2`

Reply:
Isn't the new failure because you have removed our own typing_extensions dependency that was added in #610?

Reply:
Ah yes maybe. Sorry I'm mid-hackday so probably missed that

Reply:
I was very confused

Reply:
Reinstored, let's see

Reply:
Passing now @ntessore